application/src/objects/car-sprite.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var Phaser       = require('phaser');
+var rotateVector = require('../util').rotateVector;
+
+var Car = function(state, x, y, key, group)
+{
+    Phaser.Sprite.apply(this, [state.game, x, y, key, group]);
+
+    state.game.physics.p2.enable(this);
+
+    this.body.mass = 10;
+};
+
+Car.prototype = Object.create(Phaser.Sprite.prototype);
+
+Car.prototype.updateWithinState = function(state)
+{
+    this.body.setZeroRotation();
+
+    var carRefVelocity = rotateVector(-this.body.rotation, [this.body.velocity.x, this.body.velocity.y]);
+
+    // apply rolling friction
+    this.body.applyForce(
+        rotateVector(this.body.rotation, [0, carRefVelocity[1] * 0.175 * this.body.mass]),
```

Comment:
I was thinking we could set all these hard-coded values (`0.175`) that control things like acceleration force, friction and turn speed to object properties so that we could change them on different cars without having to rewrite the update functions.